docs/tutorial/index.rst

```diff
@@ -173,7 +173,7 @@ Checking the first build
 Read the Docs will try to build the documentation of your project
 right after you create it.
 To see the build logs,
-click on the "Your documentation is building" link on the dashboard,
+click on the "Your documentation is building" link on the project home,
```

Comment:
This could link to the glossary definition. Hopefully w/ hoverxref it'll define it on hover 🤯

Reply:
Done!